editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1220,7 +1220,7 @@ void AnimationPlayerEditor::_onion_skinning_menu(int p_option) {
 	}
 }
 
-void AnimationPlayerEditor::_unhandled_key_input(const Ref<InputEvent> &p_ev) {
+void AnimationPlayerEditor::unhandled_key_input(const Ref<InputEvent> &p_ev) {
```

Comment:
Should this be `Ref<InputEventKey`>?

Reply:
No.. at some point this was changed to also receive InputEventShurtcut too, so I updated the documentation accordingly.